### DISCUSSION TITLE
Unable to resolve actions. Repository not found

### DISCUSSION THREAD

**Discussion by jlouros** [community/community#25727] [Actions]
<p>This issue was reported this morning by our Development team.</p>
<p>We have been using this action <strong>MirageNet/nunit-reporter</strong> for the last 6 months without issues. Today GitHub Actions is throwing the following error:</p>
<p><strong>Error: Unable to resolve actions. Repository not found : mirrorng/nunit-reporter.</strong></p>
<p>This is the GitHub Workflow code we are using:</p>
<pre><code class="lang-auto">      - name: unit tests reporter
        uses: MirrorNG/nunit-reporter@v1.0.9
        if: always() # run this step even if previous step failed
        with:
          path: results/*.xml # path to test results
          access-token: ${{ secrets.GITHUB_TOKEN }}
</code></pre>
<p>However, the GitHub repository exists <a href="https://github.com/MirageNet/nunit-reporter" rel="noopener nofollow ugc">https://github.com/MirageNet/nunit-reporter</a> and the selected version also exists <a href="https://github.com/MirageNet/nunit-reporter/releases/tag/v1.0.9" rel="noopener nofollow ugc">https://github.com/MirageNet/nunit-reporter/releases/tag/v1.0.9</a></p>
<p>Any guidance to resolve this issue will be highly appriciated.</p>

**Comment by airtower-luna** [comment]
<aside class="quote no-group" data-username="jlouros" data-post="1" data-topic="224041">
<div class="title">
<div class="quote-controls"></div>
 jlouros:</div>
<blockquote>
<p><code>        uses: MirrorNG/nunit-reporter@v1.0.9</code></p>
</blockquote>
</aside>
<aside class="quote no-group quote-modified" data-username="jlouros" data-post="1" data-topic="224041">
<div class="title">
<div class="quote-controls"></div>
 jlouros:</div>
<blockquote>
<p>However, the GitHub repository exists <a href="https://github.com/MirageNet/nunit-reporter" class="inline-onebox">GitHub - MirageNet/nunit-reporter: Creates a github check with nunit results</a></p>
</blockquote>
</aside>
<p>The repository names don’t match. If I try to open <code>https://github.com/MirrorNG/nunit-reporter</code> I get redirected to MirageNet, so I guess the organization got renamed at some point. Try updating the name in your workflow.</p>

**Comment by dschwank** [comment, marked as answer]
<p>I also got the same error with these two actions since today:</p>
<ul>
<li>
<code>k6io/action</code>: <a href="https://github.com/k6io/action" class="inline-onebox" rel="noopener nofollow ugc">GitHub - grafana/k6-action: Open source load testing tool and SaaS for ambitious engineering teams, now available as a GitHub Action</a>
</li>
<li>
<code>technote-space/auto-cancel-redundant-job</code>: <a href="https://github.com/technote-space/auto-cancel-redundant-job" class="inline-onebox" rel="noopener nofollow ugc">GitHub - technote-space/auto-cancel-redundant-workflow: GitHub Actions to automatically cancel redundant workflow.</a>
</li>
</ul>
<p>I noticed that both are redirected. I doubt that all those are updated / moved on the same day :thinking:</p>
<p>My best guess would be, that GitHub Actions have issues with redirected repos / don’t support them anymore.</p>

**Comment by abhisek247767** [comment]
name: Issue 
on:
  issues:
    types: [opened]

jobs:
  comment:
    runs-on: ubuntu-latest

    steps:
      - name: Add comment 
        uses: actions-ecosystem/action-add-comment@v1
        with:
          github_token: ${{ secrets.GITHUB_TOKEN }}
          issue_number: ${{ github.event.issue.number }}
          comment: |
            help!!

this also tell Error: Unable to resolve action actions-ecosystem/action-add-comment, repository not found

**Reply by airtower-luna** [reply]
https://github.com/actions-ecosystem/action-add-comment returns 404, so most likely it has been deleted or made private.

**Comment by abhisek247767** [comment]
no it's public.
her is the link of repo  https://github.com/abhisek247767/LeetCode2024-6Companies30Days

**Reply by airtower-luna** [reply]
It's not about *your* repo, the one where `actions-ecosystem/action-add-comment` would need to be doesn't exist.

**Reply by abhisek247767** [reply]
can you please tell how to resolve this is my repos.. i want to do auto comment in every issue.

**Reply by airtower-luna** [reply]
You cannot use an action that doesn't exist. Either find one that exists and does what you want (make sure the source is trustworthy!), or write your own code using the GitHub API, see https://docs.github.com/en/rest/guides/working-with-comments?apiVersion=2022-11-28 for a starting point (mind that in an action/workflow you can use `github.token` for authentication).
